lib/compilers/coffee-compiler.js

```diff
@@ -8,7 +8,10 @@ module.exports = function (raw, cb, compiler) {
   try {
     compiled = coffee.compile(raw, {
       bare: true,
-      sourceMap: true
+      sourceMap: true,
+      transpile: {
+        presets: ['env']
```

Comment:
Would we want to use presets that the user has in their babelrc?

Reply:
@eddyerburgh Yes, actually that should be better